src/reflection.cpp

```diff
@@ -481,4 +481,232 @@ Offset<const Table *> CopyTable(FlatBufferBuilder &fbb,
   }
 }
 
+bool VerifyStruct(flatbuffers::Verifier &v,
+                  const flatbuffers::Table &parentTable,
+                  voffset_t fieldOffset,
+                  const reflection::Object &obj,
+                  bool isRequired) {
+  assert(obj.is_struct());
```

Comment:
as much as I like asserts, the caller is calling this function because is_struct was true, so this one is a bit redundant.